diffsync/store/__init__.py

```diff
@@ -159,6 +159,24 @@ def get_or_instantiate(
 
         return obj, created
 
+    def get_or_instantiate_model(self, obj: "DiffSyncModel") -> Tuple["DiffSyncModel", bool]:
+        """Attempt to get the object with provided obj identifiers or instantiate obj.
+
+        Args:
+            obj: An obj of the DiffSyncModel to get or create.
+
+        Returns:
+            Provides the existing or new object and whether it was created or not.
```

Comment:
This needs to be updated to reflect the functionality (created vs. instantiated vs. added).